changelog

```diff
@@ -23,7 +41,7 @@ turnkey-ansible-15.1 (1) turnkey; urgency=low
   * Include Debian 'python3-pyfits' package. Not directly required but
     includes a working weakref.py file. The Python3 version in Stretch is
     buggy - see Debian bug #918716. Closes #1324.
-    Issue #1325 tracks the future removal of the unrequired dependency at some
+    Issue #1325 tracks the future removal of the unrequited dependency at some
```

Comment:
Is this change intentional?

Reply:
Haha! Great catch, no it wasn't... I'll fix that! :smile: